viewing_party/party.py

```diff
@@ -0,0 +1,145 @@
+# WAVE 1
+def create_movie(title, genre, rating):
+    if bool(title) and bool(genre) and bool(rating):
+        new_movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return new_movie
+    else:
+        return None
+
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+        print(user_data) 
+    else:
+        return user_data
```

Comment:
Be careful about modifying a collection as you're iterating through it (watchlist). For this example, we're only removing a single instance of a movie, however, if there are multiple instances of that movie that could cause the rest of the items in the list to shift position and skip over other items as the loop continues.

This case is fine, but let's proceed with caution when modifying a collection while iterating through it.

Also, nice `for else` loop!! `else` clauses in loops is a  unique _python_ quirk, so just be aware of that when learning other languages. Omitting the else clause and returning `user_data` outside of the loop will achieve the same goal.

```suggestion
    for movie in user_data["watchlist"]:
        if movie["title"] == title:
            user_data["watchlist"].remove(movie)
            user_data["watched"].append(movie)
     
    return user_data
```

Also, I recommend that you remove print statements or comment them out with a description of how you'll use it later.